packages/backend/src/scheduler/SchedulerWorker.ts

```diff
@@ -203,6 +204,13 @@ export class SchedulerWorker {
                         payload,
                     );
                 },
+                validateProject: async (payload: any, helpers: JobHelpers) => {
```

Comment:
```ts
                validateProject: async (payload: any, helpers: JobHelpers) => {
```
any way we can type this `payload?

Reply:
mmm, I don't think so, the last time I tried it was not possible, this is a graphile thing